sdk/lib/cmlib/hivewrt.c

```diff
@@ -129,8 +129,11 @@ HvpWriteLog(
     /*
      * Now calculate the bitmap and buffer sizes to hold up our
      * contents in a buffer.
+     *
+     * George: Please fix this properly! Either the dirty vector is a bitmap
```

Comment:
The dirty vector is indeed a bitmap (from all the documentation about registry hive file format https://reactos.org/wiki/Registry )

Reply:
Yeah, but one bit per sector, while the one that is enumerated is per 4096 bytes. This wasn't really clear. I'll just do the allocation alone to quick-fix the pool corruption and let Thomas or George fix the rest.